apps/wallet-dashboard/app/(protected)/staking/page.tsx

```diff
@@ -44,48 +62,103 @@ function StakingDashboardPage(): JSX.Element {
+                        label="Your stake"
+                        value={totalDelegatedStakeFormatted}
+                        supportingLabel={symbol}
+                    />
+                    <DisplayStats
+                        label="Earned"
+                        value={totalDelegatedRewardsFormatted}
+                        supportingLabel={symbol}
+                    />
+                </div>
+                <Title title="In progress" size={TitleSize.Small} />
+                <div className="flex max-h-[420px] w-full flex-1 flex-col items-start overflow-auto">
+                    {hasInactiveValidatorDelegation ? (
+                        <div className="mb-3">
+                            <InfoBox
+                                type={InfoBoxType.Default}
+                                title="Earn with active validators"
+                                supportingText="Unstake IOTA from the inactive validators and stake on an active validator to start earning rewards again."
+                                icon={<Info />}
+                                style={InfoBoxStyle.Elevated}
                             />
-                        ))}
+                        </div>
+                    ) : null}
+                    <div className="w-full gap-2">
+                        {system &&
+                            delegations
+                                ?.filter(({ inactiveValidator }) => inactiveValidator)
+                                .map((delegation) => (
+                                    <StakingCard
+                                        extendedStake={delegation}
+                                        currentEpoch={Number(system.epoch)}
+                                        key={delegation.stakedIotaId}
+                                        inactiveValidator
+                                        onClick={() => viewStakeDetails(delegation)}
+                                    />
+                                ))}
+                    </div>
+                    <div className="w-full gap-2">
+                        {system &&
+                            delegations
+                                ?.filter(({ inactiveValidator }) => !inactiveValidator)
+                                .map((delegation) => (
+                                    <StakingCard
+                                        extendedStake={delegation}
+                                        currentEpoch={Number(system.epoch)}
+                                        key={delegation.stakedIotaId}
+                                        onClick={() => viewStakeDetails(delegation)}
+                                    />
+                                ))}
                     </div>
-                </Box>
-                <Button onClick={handleNewStake}>New Stake</Button>
+                </div>
             </div>
             {isDialogStakeOpen && stakeDialogView && (
```

Comment:
it seems like something's going on here, as the dialog isn't opening, and instead, a 0 is showing up :see_no_evil: 
![image](https://github.com/user-attachments/assets/fe075a29-3735-4913-8c07-8aabbd751516)